src/validators/v1/issues.js

```diff
@@ -6,5 +6,8 @@ module.exports = {
 			.withMessage('description field is empty')
 			.isString()
 			.withMessage('description is invalid')
+			.not()
+			.matches(/(\b)(on\S+)(\s*)=|javascript:|<(|\/|[^\/>][^>]+|\/[^>][^>]+)>/gi)
```

Comment:
@nevil-mathew what all things it will reject here

Reply:
is a standerd regex ?

Reply:
@rakeshSgr please refer this
https://www.regextester.com/110397